### DISCUSSION TITLE
Question: Can we get a markdown page with a visual example of every theme?

### DISCUSSION THREAD

**Discussion by shmup** [rajasegar/alacritty-themes#176] [Q&A]
Similar to this:

https://github.com/eendroroy/alacritty-theme/blob/master/README.md

Reason is: using the tool is cool of course, but it's also a very step-by-step process, not to mention you might be in the mood for a light theme, but you'll be navigating through both dark and light just going through the tool's list.

**Comment by JuanVqz** [comment]
@shmup Good call!
Yes, it's worth having it! 🚀
You are pretty welcome to open a PR adding them as initial engagement.

**Reply by shmup** [reply]
@JuanVqz I think it needs to be automated, let me think about this :)
